src/Razor/src/Microsoft.VisualStudio.RazorExtension/language-configuration.json

```diff
@@ -22,7 +22,7 @@
     { "open": "<", "close": ">" }
   ],
   "indentationRules": {
-    "increaseIndentPattern": "(?:<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))(\\w[\\w\\d]*)([^/>]*(?!/)>)[^<]*$)|(?:\\{\\s*$)",
+    "increaseIndentPattern": "(?:\\s<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))(\\w[\\w\\d]*)([^/>]*(?!/)>)\\s*$)|(?:\\{\\s*$)",
```

Comment:
Could you go into the specifics of why allowing whitespace to match fixes this?

Reply:
Sure, so in C# generics such as `List<string>` there's rarely whitespace between the two statements. Meaning enforcing whitespace results in the `List` identifier invalidating the regex. Similarly with the ending whitespace requirement it ends up ruling out oddly formed HTML such as:

```HTML
<strong>Hello World|
```

or normal C# such as:
```C#
new List<string>()
```

Given the HTML case is a little more likely, if you were to hit "enter" where would you expect your cursor? There isn't a great answer so in this odd case because we enforce whitespace until the end of line our "increase indent" behavior ignores this strong tag. resulting in:
```HTML
<strong>Hello World
|
```

Versus before you'd get:
```HTML
<strong>Hello World
    |
```

It's not 100% fool proof because technically users who do:
```C#
new SomeCSharpClass    <string>
{
}
```
would get funky indentation but I'm alright with that given we're trying to respect the most common case.